packages/client/src/rpc/modules/engine.ts

```diff
@@ -1231,12 +1231,6 @@ export class Engine {
         }
       }
       this.service.txPool.removeNewBlockTxs(blocks)
-
-      const isPrevSynced = this.chain.config.synchronized
-      this.config.updateSynchronizedState(headBlock.header)
-      if (!isPrevSynced && this.chain.config.synchronized) {
-        this.service.txPool.checkRunState()
-      }
     } else if (!headBlock.isGenesis()) {
```

Comment:
What does removing this section do?

Reply:
it seems to be a redundant txpool start snippet, just below this if/else there is a similar snippet